onset_maproom/calc.py

```diff
@@ -400,58 +389,52 @@ def seasonal_onset_date(
-        dry_spell_length : int
-                    Length in days of dry spell that would invalidate a wet spell as onset date 
-                    if found within the 'dry_spell_search' days following the 'wet_spell_length'-day window 
-                    that met the wet spell criteria.      
-        dry_spell_search : int
-                    Length in days to search for a 'dry_spell_length'-day dry spell after a wet spell 
-                    is found that would invalidate the wet spell as onset date. 
-        time_coord : str, optional
-                    Time grid in 'soil_moisture' (the default is "T", which implies
-                    naming convention of time grid as "T" in 'soil_moisture')
-        Returns
-        -------
-        seasonal_cess_date : Dataset
-                    Dataset containing days since search start date as timedelta,
-                    and cessation date as datetime for each year in soil moisture DataArray.
-        See Also
-        --------
-        Notes
-        -----
-        Examples
-        --------
-
+    """Use daily soil moisture data to compute yearly seasonal onset dates from daily rainfall.
+
+    Compute yearly dates by utilizing groupby function to group data by season 
+    and onset_date function to calculate onset date for each year of grouped data.
+
+    Parameters
+    ----------
+    daily_rain : DataArray
+        Array of daily rainfall values.
+    search_start_day : int
+        The day part (1-31) of the date to start scanning for onset date.
+    search_start_month : int
+        The month part (1-12) of the date to start scanning for onset date.
+    search_days : int
+        Number of days from search start date to scan for onset date.
+    wet_thresh : int
+        Rainfall threshold to determine wet day.
+    wet_spell_length : int
+        Length in days of running window when `wet_thresh` is to be met to define a wet spell.
+    wet_spell_thresh : int
+        Threshold of rainfall to be reached during `wet_spell_length` window to define a wet spell. 
+    min_wet_days : int
+        Minimum number of wet days in `wet_spell_length` window when it rained at or above 
+        `wet_spell_thresh` to be considered a wet spell.
+    dry_spell_length : int
+        Length in days of dry spell that would invalidate a wet spell as onset date 
+        if found within the `dry_spell_search` days following the `wet_spell_length`-day window 
+        that met the wet spell criteria.      
+    dry_spell_search : int
+        Length in days to search for a `dry_spell_length`-day dry spell after a wet spell 
+        is found that would invalidate the wet spell as onset date. 
+    time_coord : str, optional
+        Time grid in `soil_moisture` (default time_coord='T').
+    Returns
+    -------
+    seasonal_onset_date : Dataset
+        Dataset containing days since search start date as timedelta,
+        and onset date as datetime for each year in `daily_rain`.
```

Comment:
I don't think this is correct. Maybe I'll revisit after this PR?

Reply:
yes that would be useful unless you can just type what it should be here and I add it